tests/Fixtures/Scratch/NullRef31.yaml

```diff
@@ -23,11 +23,12 @@ paths:
           description: 'Ref plus response'
           content:
             application/json:
+              example: 'Some example about the content'
               schema:
                 oneOf:
-                  - { $ref: '#/components/schemas/repository', description: 'The repository' }
```

Comment:
https://github.com/DjordyKoert/swagger-php/blob/2eedbb3a9693762720a7aff34d6a2af8303bc593/src/Annotations/AbstractAnnotation.php#L358

`$ref` no longer exists after going through the OpenApi31Processor because it gets set to `Generator::UNDEFINED` which makes it no longer copy the description.

I have another work in progress branch to test if a new annotation `Reference` can be implemented for this with instead. https://github.com/DjordyKoert/swagger-php/blob/feat/reference-object-annotation/src/Annotations/Reference.php